dashboard/app/assets/javascripts/lti/early_access_banner.js

```diff
@@ -0,0 +1,66 @@
+class LtiEarlyAccessBannerHandler {
```

Comment:
Is it possible to move this new JS file into the /apps directory with the majority of the other Javascript files? If I run the yarn linter on this file, it comes back with some issues (although they are all just preferring single quotes over double):
```
   3:21  error  Replace `"#lti-early-access-banner"` with `'#lti-early-access-banner'`                prettier/prettier
   8:57  error  Replace `"key"` with `'key'`                                                          prettier/prettier
  10:43  error  Replace `"closed"` with `'closed'`                                                    prettier/prettier
  18:22  error  Replace `".close").on("click"` with `'.close').on('click'`                            prettier/prettier
  20:45  error  Replace `"closed"` with `'closed'`                                                    prettier/prettier
  26:7   error  Replace `"#codeApp"` with `'#codeApp'`                                                prettier/prettier
  28:24  error  Replace `".close").addClass("no-mc"` with `'.close').addClass('no-mc'`                prettier/prettier
  32:9   error  Replace `"#codeApp"` with `'#codeApp'`                                                prettier/prettier
  38:35  error  Replace `"#codeApp"` with `'#codeApp'`                                                prettier/prettier
  40:29  error  Replace `"class"` with `'class'`                                                      prettier/prettier
  48:23  error  Replace `"#codeApp"` with `'#codeApp'`                                                prettier/prettier
  49:21  error  Replace `"position")·!==·"absolute"` with `'position')·!==·'absolute'`                prettier/prettier
  52:30  error  Replace `"top"` with `'top'`                                                          prettier/prettier
  54:45  error  Replace `"top"` with `'top'`                                                          prettier/prettier
  57:42  error  Replace `":visible"` with `':visible'`                                                prettier/prettier
  61:30  error  Replace `"top",·newCodeAppTop,·"important"` with `'top',·newCodeAppTop,·'important'`  prettier/prettier
  62:36  error  Replace `"resize"` with `'resize'`                                                    prettier/prettier
  66:3   error  Replace `"#lti-early-access-banner"` with `'#lti-early-access-banner'`                prettier/prettier

✖ 18 problems (18 errors, 0 warnings)
  18 errors and 0 warnings potentially fixable with the `--fix` option.
```

Reply:
I've just replaced single quotes with double quotes because the linter told me to do so before creating the commit 🙃

Reply:
It's possible, I'm just not sure if `/app` is the right place for the `dashboard`-specific scripts.

Reply:
@daynew for a script like this that's just tied to a single haml page, is /apps still the best place for it? My gut is that we put dashboard-only react components in there, so javascript would also go there.